### PR TITLE
Using `large` VMs travis

### PR DESCRIPTION
**Pull request by TotoGaz**
If this succeeds, let's see if we can find define one size per job (CUDA builds get more).

### CODE REVIEW

**Comment by rrsettgast** [comment]
@TotoGaz Let's just merge this. It is what we are paying for anyways.

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +19,8 @@ geosx_osx_build: &geosx_osx_build
   os: osx
   osx_image: xcode13.2
   install:
+  - echo "running nproc"; nproc
+  - top -l 1 -s 0 | grep PhysMem
```
Comment (TotoGaz) [inline .travis.yml:23]: Do we need those lines merged?

**Comment by TotoGaz** [comment]
I cannot approve my own PR 🤣

**Review by bmhan12 — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (bmhan12) [reply .travis.yml:23]: I'm fine, either way. We can keep it for debugging purposes or remove for cleanliness.

Comment (TotoGaz) [reply .travis.yml:23]: I'll think we'll find the cleaner way to get this information after all our final moves.

You're next in line in the merge queue for both incoming PRs.
Can you merge? Consider the order you think most appropriate for merging 😉

Comment (bmhan12) [reply .travis.yml:23]: > Can you merge? Consider the order...

I do not have merge authority.
Feel free to merge this PR first.

**Pull request merged**